config.json

```diff
@@ -618,7 +618,38 @@
       "name": "Type Conversion"
     }
   ],
-  "key_features": [],
+  "key_features": [
+    {
+      icon: "web",
+      title: "Popular for Web and Mobile Development",
+      content: "Dart has the Flutter library, which can be used to quickly build web and smartphone applications."
+    },
+    {
+      icon: "cross-platform",
+      title: "Instant iOS and Android Development",
+      content: "Dart/Flutter can be converted into iOS's Swift language and Android's Kotlin language, making mobile development fast and efficient."
+    },
+    {
+      icon: "community",
+      title: "Full Documentation",
+      contents: "Dart's documentation and learning materials are constantly updated by its developers, making it easy to look up information on the language."
```

Comment:
I think this is a great inclusion. This would be a good spot to point to `api.dart.dev` so folks can see the documentation in full.

Reply:
The key features aren't meant purely to entice people into trying the language. We have only 100 characters to play with, so we'll have to leave the link to the api docs out.